pyvrp/cpp/search/LocalSearch.cpp

```diff
@@ -282,6 +282,9 @@ void LocalSearch::update(Route *U, Route *V)
 
 void LocalSearch::loadSolution(Solution const &solution)
 {
+    if (!solution.isComplete())  // TODO allow incomplete at some point
```

Comment:
I don't mind but I thought you preferred issues over TODO's in the code?

Reply:
#307!